src/nunavut/templates.py

```diff
@@ -0,0 +1,120 @@
+#
+# Copyright 2019 Amazon.com, Inc. or its affiliates. All Rights Reserved.
+# Copyright (C) 2018-2019  UAVCAN Development Team  <uavcan.org>
+# This software is distributed under the terms of the MIT License.
+#
+"""
+Abstractions around template engine internals.
+"""
+import typing
+
+EnvironmentFilterAttributeName = 'environmentfilter'
```

Comment:
Per PEP8, this should be named in `SCREAMING_SNAKE_CASE`. Why is it different here?

Reply:
REALLY? MY_PEP8_LINTER_DIDN'T_COMPLAIN. LET_ME_LOOK_INTO_THIS!

Reply:
PEP8 says:

> Constants are usually defined on a module level and written in all capital letters with underscores separating words. Examples include MAX_OVERFLOW and TOTAL.